src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<ValidPassword, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto user,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return user.getPassword().equals(user.getRepeatPassword());
```

Comment:
What will happen if you get password == null ?

Reply:
I added check for null, but I thought, if we have @NotNull in our requestDto, we will have BAD REQUEST.